### PR TITLE
ABI documentation clarifications

### PR DESCRIPTION
**Pull request by Marenz**

### CODE REVIEW

**Review by ekpyron — DISMISSED** [review]
Looks good to me!

File: docs/abi-spec.rst [hunk 1]
```diff
@@ -141,19 +141,19 @@ on the type of ``X`` being
...
   where ``X = (X(1), ..., X(k))`` and
-  ``head`` and ``tail`` are defined for ``Ti`` being a static type as
+  ``head`` and ``tail`` are defined for ``Ti`` as follows:
+
+  if ``Ti`` is **static**:
 
     ``head(X(i)) = enc(X(i))`` and ``tail(X(i)) = ""`` (the empty string)
 
-  and as
+  otherwise if ``Ti`` is **dynamic**:
```
Comment (chriseth) [inline docs/abi-spec.rst]:
```suggestion
  otherwise, i.e. if ``Ti`` is **dynamic**:
```

File: docs/abi-spec.rst [hunk 2]
```diff
@@ -141,19 +141,19 @@ on the type of ``X`` being
   ``enc(X) = head(X(1)) ... head(X(k)) tail(X(1)) ... tail(X(k))``
 
   where ``X = (X(1), ..., X(k))`` and
-  ``head`` and ``tail`` are defined for ``Ti`` being a static type as
+  ``head`` and ``tail`` are defined for ``Ti`` as follows:
+
+  if ``Ti`` is **static**:
```
Comment (chriseth) [inline docs/abi-spec.rst]: Why italics (or is it even bold)?

Comment (Marenz) [reply docs/abi-spec.rst]: It's **bold**. I felt like that word is the most important part in the sentence and deserved an emphases for fast and easy recognition

**Comment by Marenz** [comment]
Updated

**Review by ekpyron — DISMISSED** [review]

File: docs/abi-spec.rst (hunk 2 above)
Comment (chriseth) [reply docs/abi-spec.rst]: Nothing else is bold anywhere and the sentence is short enough such that you immediately notice that `Ti` and `static` are the two most important parts in this sentence.

Comment (Marenz) [reply docs/abi-spec.rst]: > Nothing else is bold anywhere 

That's not necessarily a good thing :P Maybe more boldness would make it easier to read everywhere.

I dunno, I felt it reads nicer that way. But if you're against it, I'll remove it again.

Comment (chriseth) [reply docs/abi-spec.rst]: I think boldness distracts. Only headings should be bold. If you want to emphasize something, make it in italics, so you notice while you read it but not while you are looking at something else.

Comment (ekpyron) [reply docs/abi-spec.rst]: I'm fine with it either way :-).

**Comment by Marenz** [comment]
Unboldified.

**Review by chriseth — APPROVED** [review]
Thanks!

**Pull request merged**